airflow/www/security.py

```diff
@@ -630,7 +630,7 @@ def sync_perm_for_dag(self, dag_id, access_control=None):
         :type dag_id: str
         :param access_control: a dict where each key is a rolename and
             each value is a set() of permission names (e.g.,
-            {'can_read'}
+            {permissions.ACTION_CAN_READ}
```

Comment:
```suggestion
            {airflow.security.permissions.ACTION_CAN_READ}
```

I think, so it's clearer from the docs where this constant is.

Reply:
This is actually the result of a bad search and replace. I wasn't trying make comment strings use the constants. Do you think they should though?

Reply:
Could go either way. For user code/documentation it's probably easier to use string literals.